lib/mrsk/utils/healthcheck_poller.rb

```diff
@@ -1,5 +1,5 @@
 class Mrsk::Utils::HealthcheckPoller
-  TRAEFIK_HEALTHY_DELAY = 1
+  TRAEFIK_HEALTHY_DELAY = 2
```

Comment:
@kevinmcconnell - the integration tests were consistently failing in Github actions until I upped this to 2 seconds. Traefik was returning a 404 when we checked it the app was up.

Worked fine with 1 second on my M1, so probably a resource constraint issue - I think its worth sticking with a slightly longer default delay, but maybe this should be configurable.

Reply:
Thanks, and I agree, it would be worth making this configurable.